commit 3d59c6f7d83f54d5c73013de5ab61e9a0ecd1460
Author: Remi Gacogne <remi.gacogne@powerdns.com>
Date:   Mon Oct 17 10:07:26 2016 +0200

    Fix building with ECDSA support disabled in libcrypto
    
    (cherry picked from commit aa74d164ae29269168d048d2cc8d7e1f984774c4)

diff --git a/opensslsigners.cc b/opensslsigners.cc
index 9e919a0..acf8f08 100644
--- a/opensslsigners.cc
+++ b/opensslsigners.cc
@@ -128,6 +128,7 @@ static inline int RSA_set0_crt_params(RSA* rsakey, BIGNUM* dmp1, BIGNUM* dmq1, B
   return 1;
 }
 
+#ifdef HAVE_LIBCRYPTO_ECDSA
 static inline void ECDSA_SIG_get0(const ECDSA_SIG* signature, const BIGNUM** pr, const BIGNUM** ps) {
   *pr = signature->r;
   *ps = signature->s;
@@ -140,6 +141,8 @@ static inline int ECDSA_SIG_set0(ECDSA_SIG* signature, BIGNUM* pr, BIGNUM* ps) {
   signature->s = ps;
   return 1;
 }
+#endif /* HAVE_LIBCRYPTO_ECDSA */
+
 #else
 void openssl_thread_setup() {}
 void openssl_thread_cleanup() {}
